zoobc/zoobc-core
# update genesis participation score to the max score

## Description
By @bzpython request, The genesis participation score is updated to the max score.

## Breakdown
- [x] Update the constant

## Reference Issue
Close #numberIssue

## Step to Test (optional)
- Run a fresh node
- observe that the nodes are having maximum score at the beginning
